Add hackathon doc to blog (#14615)
PostHog/posthog.com: contents/blog/behind-the-scenes-posthog-hackathon.md

---
date: 2026-01-28
title: Behind the scenes of a PostHog hackathon
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - andy-vandervell
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/ben_dreams_of_hedgehogs_a90e7c58cf.png
featuredImageType: full
category: Inside PostHog
tags:
  - Offsites
---

As a 100% remote company, our annual company hackathon is one of our most important rituals.  

> Some of the most important features, like session replay and our data warehouse, were the direct results of hackathons.

Author: ```suggestion
Some of the most important features, like session replay and our data warehouse, started as hackathon projects.
```
